src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball() {
+
+    }
```

Comment:
empty constructor is created by default, if there are no other constructors declared